src/TwigComponent/src/Twig/ComponentExtension.php

```diff
@@ -42,7 +43,15 @@ public function getFunctions(): array
     {
         return [
             new TwigFunction('component', [$this, 'render'], ['is_safe' => ['all']]),
-            new TwigFunction('cva', [$this, 'cva']),
+            new TwigFunction('cva', [$this, 'cva'], [
+                'deprecation_info' => new DeprecatedCallableInfo(
```

Comment:
This won't be usable by anyone not perfectly up to date.. and i don't want to force people upgrade for a deprecation :/ 

Can't we use the other syntax ? Or just trigger in the method ?

Reply:
That was my feeling too, but this new deprecation feature was legit developed for us :p 

But you're right, I'm not a fan of forcing a Twig update to display a deprecation message more appropriately. Let's do it simple

Reply:
Well, I think we don't really have the choice in fact, since options `deprecated`, `alternative` are deprecated in favor of `deprecation_info`, see checks for TwigComponent, LiveComponent, Map, etc...

Those deprecations will appear to the end user... right? 
I think we will have to wait for a new Twig release

Reply:
No no no :) 

You cannot assume users will have the new Twig version installed. 

One way could be to check the existence of the new DeprecatedCallableInfo class and act accordingly .. wdyt ?